xbmc/video/test/TestVideoFileItemClassify.cpp

```diff
@@ -196,6 +196,13 @@ TEST(TestVideoFileItemClassify, IsSubtitle)
   EXPECT_FALSE(VIDEO::IsSubtitle(CFileItem("random.notasub", false)));
 }
 
+TEST(TestVideoFileItemClassify, IsVideoAssetsFile)
+{
+  EXPECT_TRUE(VIDEO::IsVideoAssetFile(CFileItem("videodb://foo/bar?videoversionid=1", false)));
+  EXPECT_FALSE(VIDEO::IsVideoAssetFile(CFileItem("videodb://foo/bar?videoversionid=1", true)));
+  EXPECT_FALSE(VIDEO::IsVideoAssetFile(CFileItem("videodb://foo/bar", false)));
```

Comment:
A few additional tests if you don't mind.

An actual url that should succeed is a file with url "videodb://movies/videoversions/40402?mediaid=8&videoversionid=40402"

However similar url as folder (m_bIsFolder == true) should be rejected "videodb://movies/videoversions/0?mediaid=8&videoversionid=0" - it represents the movie as a folder in the library

Reply:
that is exactly what I tried to cover with my entries, so I replaced rather than add.